### ISSUE TITLE
No main manifest attribute in .jar

### ISSUE THREAD

**Issue by madhurtewani** [dkpro/dkpro-jwpl#205]
Hi, I downloaded jar file for DataMachine from [this link](https://search.maven.org/search?q=de.tudarmstadt.ukp.wikipedia).

I have downloaded 3 files (pages-articles, pagelinks, categorylinks) from [this link](https://dumps.wikimedia.org/enwiki/latest/) for data samples as mentioned [here](https://dkpro.github.io/dkpro-jwpl/DataMachine/).

After downloading jar file and data samples when I try to run jar on data samples, I get following error.
`no main manifest attribute, in .\JWPLDataMachine.jar`
Just FYI, I do see a `MANIFEST.MF` file under `JWPLDataMachine.jar\META-INF`.

The command that I have run with all the parameters is mentioned below.
```
java -jar .\JWPLDataMachine.jar english People Disambiguation .\source_files
```
and
```
java -jar .\JWPLDataMachine.jar de.tudarmstadt.ukp.wikipedia.datamachine.domain.JWPLDataM
achine english People Disambiguation .\source_files
```

Folder `source_files` contains the 3 files for data samples.

Can you please help me with this problem?

**Comment by zesch** [comment]
Which version are you using?

-Torsten

﻿On 19.07.19, 13:54, "Madhur Tewani" <notifications@github.com> wrote:

    Hi, I downloaded jar file for DataMachine from 
    this link <https://search.maven.org/search?q=de.tudarmstadt.ukp.wikipedia>.
    I have downloaded 3 files (pages-articles, pagelinks, categorylinks) from 
    this link <https://dumps.wikimedia.org/enwiki/latest/> for data samples as mentioned 
    here <https://dkpro.github.io/dkpro-jwpl/DataMachine/>.
    After downloading jar file and data samples when I try to run jar on data samples, I get following error.
    no main manifest attribute, in .\JWPLDataMachine.jar
    Just FYI, I do see a MANIFEST.MF file under JWPLDataMachine.jar\META-INF.
    The command that I have run with all the parameters is mentioned below.
    java -jar .\JWPLDataMachine.jar english People Disambiguation .\source_files
    and
    java -jar .\JWPLDataMachine.jar de.tudarmstadt.ukp.wikipedia.datamachine.domain.JWPLDataM
    achine english People Disambiguation .\source_files
    Folder source_files contains the 3 files for data samples.
    Can you please help me with this problem?
    —
    You are receiving this because you are subscribed to this thread.
    Reply to this email directly, 
    view it on GitHub <https://github.com/dkpro/dkpro-jwpl/issues/205?email_source=notifications&email_token=AAURBYEP3WQV5ZNELKFW4ZLQAGTN3A5CNFSM4IFFHRCKYY3PNVWWK3TUL52HS4DFUVEXG43VMWVGG33NNVSW45C7NFSM4HAIAELQ>, or

**Comment by madhurtewani** [comment]
I am using the latest version as per [this link](https://search.maven.org/search?q=de.tudarmstadt.ukp.wikipedia) i.e. 1.1.0.

**Comment by zesch** [comment]
If you want to run it from the command line, you should try the jar-with-dependencies.jar version that is only available up to version 1.0.0

**Comment by reckart** [comment]
Assuming the answer resolves the issue.
